packages/consumption/src/modules/common/ValidationResult.ts

```diff
@@ -21,7 +21,13 @@ export abstract class ValidationResult {
 
     public static fromItems(items: ValidationResult[]): ValidationResult {
         return items.some((r) => r.isError())
-            ? ValidationResult.error(new ApplicationError("inheritedFromItem", "Some child items have errors."), items)
+            ? ValidationResult.error(
+                  new ApplicationError(
+                      "error.consumption.validation.inheritedFromItem",
+                      "Some child items have errors. If this error occured during the creation of a request, we recommand to call 'validate' to get more information."
```

Comment:
i think there is no "validate". It's called canCreate

Reply:
also I would not talk about "us". I'd suggest `If this error occurred during the creation of a request, call 'canCreate' to get more information.`

Reply:
But you have to post /api/v2/Requests/Outgoing/Validate and then canCreate will be used or isn't so?
https://enmeshed.eu/use-case-consumption-check-if-outgoing-request-can-be-created

Reply:
true, the connector route is called like that - but we are in the runtime and should stick to runtime terminology

Reply:
Just in case you keep a similar wording: It's _recomm**e**nd_, not _recomm**a**nd_.